### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by kr-sarvesh**
please review my product page

https://sharp-keller-3938db.netlify.app/screens/product/product.html

link added above

### CODE REVIEW

**Comment by TJain02** [comment]
Amazing UI. Even the code is clean. Looks good to merge.

**Review by TJain02 — APPROVED** [review]

File: screens/product/product.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.title {
+  text-align: center;
+  margin: 0 auto 80px;
+  position: relative;
+  line-height: 60px;
+  color: var(--primary-color);
+}
+
+h4 {
```
Comment (athar05) [inline screens/product/product.css:34]: This can be a utility class

**Review by athar05 — COMMENTED** [review]
Overall looks good

**Comment by athar05** [comment]
Product listing page needs to have a filter section

**Comment by SuchitaGhadge** [comment]
looks good to merge

**Pull request merged**